system-tests/README.md

```diff
@@ -1,6 +1,6 @@
 # System tests
 
-The system tests copies a file from a provider to a consumer blob storage account.
+The system tests copies a file from a provider (`company1`) to a consumer (`company2`) blob storage account.
```

Comment:
grammar "tests copies". In fact should be reformulated since there are other tests too

Reply:
Updated by @algattik. Thanks for the help. Resolving it.